.codespellrc

```diff
@@ -0,0 +1,3 @@
+[codespell]
+skip = googletest-release*,autom4te.cache,./service/docs/build,./service/contrib,libtool*,ltmain*,./integration/apps/*/*,./.git,./ChangeLog,*.patch,./test/ProfileTableTest.cpp,configure,config.*,depcomp,*.spec
+ignore-words-list = atleast,clos,fom,warmup,affinitize,affinitized,whis
```

Comment:
I guess atleast and whis are okay?  Are these referring to code?

Reply:
Yeah, we use `atleast` in some test code and `whis` in box-and-whisker plotting code. Both names are provided by external packages so we can't rename them.